From 908a16d5dde09b386cd964f4c28895e990f57c1d Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 25 Jan 2021 16:34:58 +0200
Subject: [PATCH] [SW-0] ppboot: make the failure msg more precise

The failure message in the hl_enter_recovery() function is not precise
as we can get to run this function from various reasons, but it only
prints one specific reason - when no boot descriptors were found.
Rephrase the message to be more precise and reflect the situation.

Change-Id: If4faa5dd2e8ccb7d74d9d956c2e24100e9c46491
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/ppboot/ppboot.c b/ppboot/ppboot.c
index 3b3f471..5b43b71 100644
--- a/ppboot/ppboot.c
+++ b/ppboot/ppboot.c
@@ -231,7 +231,7 @@
 
 __weak void hl_enter_recovery(void)
 {
-	preboot_puts("No boot descriptors detected, so WFE\n");
+	preboot_puts("No valid images found, so WFE\n");
 	wfe();
 }
 
